applications/cere-ddc-updates.md

```diff
@@ -0,0 +1,107 @@
+
+## Team 🧑‍🤝‍🧑
+
+### Team Members
+
+-   Nikhil Bhintade
+-   Ashish Bhintade
+
+### Contact
+
+-   **Contact Name:** Nikhil Bhintade
+-   **Contact Email:** bhintadenik@gmail.com
+-   **Website:** https://linktr.ee/nikbhintade
+
+### Team's Experience
+
+**Nikhil Bhintade**: I have been working as a freelance Solidity developer and technical writer since late 2020 in the crypto industry. My development experience mainly involves forking DeFi protocols on various EVM-compatible chains, but I’ve recently focused more on creating technical content.
+
+**Ashish Bhintade**: A frontend developer who has primarily worked on frontend apps that interact with EVM chains, mainly using Next.js/React, along with libraries to interact with smart contracts and various wallets.
+
+### Team Code Repos
+
+-   N/A
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to other hosted or live projects are also acceptable.
+
+-   https://github.com/nikbhintade
+-   https://github.com/ashishbhintade
+
+### Team LinkedIn Profiles (if available)
+
+-   https://www.linkedin.com/in/nikbhintade
+-   https://www.linkedin.com/in/ashishbhintade
+
+## Development Roadmap 🔩
+
+### Overview
+
+There’s not much to explain in the roadmap, as we will be following the provided specifications. It will take us 8 days to develop a working and well-tested smart contract deployed to mainnet. The remaining time will be spent implementing automatic DDC updates.
+
+For the first bounty (swap smart contract), the following features will be included:
+
+-   On the DDC, users can connect a wallet (any EOA)
+-   After the user sends ETH/ERC-20 tokens to the contract:
+    -   The contract swaps the given token to CERE
+    -   Then bridges those tokens to the CERE mainnet using Hyperbridge
+
+Regarding the DDC, I will add more details soon—I haven’t reviewed it yet. The current focus has been on understanding Hyperbridge and the token-bridging process.
+
+### Milestone 1 — Smart Contract Functionality
+
+-   **Estimated Duration:** 10 days
+-   **FTE:** 1
+
+| Number | Deliverable    | Specification                                                                 |
+| ------ | -------------- | ----------------------------------------------------------------------------- |
+| 1      | Smart Contract | The smart contract can swap any token to CERE and bridge it using Hyperbridge |
+
+### Milestone 2 — DDC Automated Account Update
+
+-   **Estimated Duration:** 10 days
+-   **FTE:** 1
+
+| Number | Deliverable           | Specification                                                                       |
+| ------ | --------------------- | ----------------------------------------------------------------------------------- |
+| 1      | DDC Automatic Updates | Haven’t had a chance to finalize the details yet, but an outline will be ready soon |
```

Comment:
Would it be possible to add details here?

Reply:
I can add it over the weekend. Thanks! I was focused on something else so didn't get a chance to finalize it.